features/redisCustomer.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Saving and finding a customer
```

Comment:
This feature file reads more like a unit test than a behavioural test. Remember that these scenarios are intended to be something you can discuss with the business owner and have them understand and contribute to.

Putting things in the database and getting them back out are purely technical concerns. The business owner will not understand or have any input into how this works, so there is no point in showing it to them.

Of course, this stuff does need to be tested, but Behat is not the correct tool to test your Redis Repository. You should instead use PHP Spec or PHP Unit to unit test the Redis Customer Repository.